tick/stateful/eval_function_node.go

```diff
@@ -187,6 +216,12 @@ func eval(n NodeEvaluator, scope *Scope, executionState ExecutionState) (interfa
 		return n.EvalTime(scope, executionState)
 	case ast.TDuration:
 		return n.EvalDuration(scope, executionState)
+	case ast.TMissing:
```

Comment:
In summary we need EvalMissing so that a ReferenceNode can correctly return it, makes sense. There is a hacky way around that but you have done the work needed so lets leave it as it.